aries-site/src/examples/components/dateinput/DateInputExample.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box, Form, FormField, DateInput } from 'grommet';
+
+export const DateInputExample = () => {
+  const [value, setValue] = React.useState({ value: '' });
+  const onChange = nextValue => {
+    console.log('onChange', nextValue);
+    setValue(nextValue);
+  };
+  return (
+    <Box align="center" pad="large">
```

Comment:
For all of these, there is no hover behavior when I mouse over the date. It looks like it should be received `active-background`, but right now it's not changing.

Reply:
So this is due to 

```
   default: {
      color: 'text',
      border: undefined,
      padding: {
        horizontal: '12px',
        vertical: '6px',
      },
    },

```

Trying to see if there would be a way around this in `grommet` nothing sticking out so far

Reply:
Let's leave this as a separate issue filed on Grommet and also tracked in the Experience Studio backlog

Reply:
yes I will file it on grommet as well as on here.. then put my findings so far

Reply:
I'm a little late here. Is the issue here that when the Button is active={true}, there is no background style set? And we should be picking up the background defined in the theme here? https://github.com/grommet/grommet-theme-hpe/blob/d3388fed94b9bc1a9dc957d5e5477ee1b22e46f0/src/js/index.js#L303

Reply:
when we have default button defined in theme then grommet is using `buttonstyledkind` 
which was not getting the active style so thats what the PR is in grommet for